code/src/components/GameContainer.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+
+import WelcomePage from './WelcomePage'
+
+const GameBoard = () => {
+    return(
+        <>
+            <WelcomePage />
+        </>
+    )
+}
+
+export default GameBoard
```

Comment:
What is going on here? 🤔 This component has the same name as the component in file GameBoard.js, and only renders one component. We are unsure if it is in use?